api/analyzers/views.py

```diff
@@ -0,0 +1,84 @@
+import subprocess
+
+from api.models import Analyzer
+from api.models import ManageDoctorProfileAnalyzer
+
+import simplejson as json
+from django.http import JsonResponse
+
+
+def all_analyzers(request):
+    analyzers = [{"label": analyzer.title, "pk": analyzer.id} for analyzer in Analyzer.objects.all().exclude(service_name__isnull=True).exclude(port__isnull=True).order_by('title', 'id')]
+    return JsonResponse({"data": analyzers})
+
+
+def restart_analyzer(request):
+    request_data = json.loads(request.body)
+    su = request.user.is_superuser
+    if su:
+        name = Analyzer.objects.filter(id=request_data["pk"]).first()
+        restart_service = subprocess.Popen(["systemctl", "--user", "restart", name.service_name])
+        restart_service.wait()
+        result = get_status_analyzer(request_data["pk"])
+        return JsonResponse({"data": result})
+    else:
+        name = ManageDoctorProfileAnalyzer.objects.filter(analyzer_id=request_data["pk"], doctor_profile_id=request.user.doctorprofile.pk).first()
+        restart_service = subprocess.Popen(["systemctl", "--user", "restart", name.analyzer.service_name])
+        restart_service.wait()
+        result = get_status_analyzer(request_data["pk"])
+        return JsonResponse({"data": result})
+
+
+def manage_profile_analyzer(request):
+    current_user = request.user.doctorprofile.pk
+    su = request.user.is_superuser
+    if su:
+        filter_analyzer = [{"label": g.title, "pk": g.pk} for g in Analyzer.objects.all().order_by('title', 'pk')]
+        return JsonResponse({"data": filter_analyzer})
+    else:
+        filter_analyzer = [{"label": g.analyzer.title, "pk": g.analyzer_id} for g in ManageDoctorProfileAnalyzer.objects.filter(doctor_profile_id=current_user).order_by('analyzer', 'id')]
+        return JsonResponse({"data": filter_analyzer})
```

Comment:
Тут тоже чуть меньше дублирования можно
```suggestion
    return JsonResponse({"data": filter_analyzer})
```

А верхний return убрать